acloud/resource_cluster.go

```diff
@@ -195,9 +198,22 @@ func getStringAttributeWithLegacyName(d *schema.ResourceData, names ...string) s
 	return defaultValue
 }
 
+func getOrganisation(m interface{}, d *schema.ResourceData) (string, error) {
+	provider, ok := m.(ConfiguredProvider)
+
+	if !ok {
+		return "", errors.New("configured provider incorrect")
+	}
+	if provider.Organisation == "" {
+		return getStringAttributeWithLegacyName(d, "organisation", "organisation_slug"), nil
```

Comment:
What if `getStringAttributeWithLegacyName(..)` returns an empty `""` string.

That should probably be an error.